projects/saturn/src/main/java/io/fairspace/saturn/webdav/DirectoryResource.java

```diff
@@ -393,4 +377,33 @@ private void uploadMetadata(FileItem file) throws BadRequestException, ConflictE
             throw new BadRequestException("Error applying metadata", e);
         }
     }
+
+    private org.apache.jena.rdf.model.Resource getDirectory(CSVRecord record, long lineNumber) throws BadRequestException, NotAuthorizedException {
```

Comment:
Isn't it better to make the name a parameter here, instead of passing the record? It would make it reusable not only for the csv upload.